### ISSUE TITLE
WTree /htree does not always close on deselect

### ISSUE THREAD

**Issue by marksreeves** [BorderTech/wcomponents#769]
With a WTree in htree mode. CTRL/META with selection (through keyboard or mouse click) of a currently selected and open branch should

1. close the branch;
2. if the branch has a parent branch then select that parent branch's item.

Note that if item 2 is successful that it will close the original branch but if the branch being deselected does not have a parent it will not currently close.
